README.md

```diff
@@ -2,7 +2,7 @@
 This app uses [OpenWeatherMap](https://openweathermap.org/) to fetch weather data.
 ```
 Current     'http://api.openweathermap.org/data/2.5/weather?q={city}&appid={api_token}'
-Forecast    'http://api.openweathermap.org/data/2.5/forecast/daily?q={city}&cnt={days_count}&appid={api_token}'
+Forecast    'http://api.openweathermap.org/data/2.5/forecast?daily?q={city}&cnt={days_count}&appid={api_token}'
```

Comment:
I don't think this is correct – the URL will have two `?` segments:

* `?daily`
* `?q={city}&cnt={days_count}&appid={api_token}`

In addition, [the API docs say](https://openweathermap.org/forecast16#16days) it should indeed be `/data/2.5/forecast/daily` anyway...

Reply:
Forecast/ is part of 16 day paid subscription API which affects 401: Unauthorized. Thus the 5day version with forecast?. Ahaa the daily needs to be out